public/locales/fr.json

```diff
@@ -147,5 +147,7 @@
   "A profile with the same username already exists in this device, rename it and try again.": "Un profil avec le même nom existe sur cet appareil, merci de le renommer et recommencer la synchronisation.",
   "Error during the synchronization. Please retry.": "Echec de la synchronisation. Merci de réessayer.",
   "Paste profile link": "Coller le lien du profil",
-  "The link has expired. Please retry with a fresh link.": "Le lien a expiré, merci de réessayer avec nouveau lien."
+  "The link has expired. Please retry with a fresh link.": "Le lien a expiré, merci de réessayer avec nouveau lien.",
+  "Scanner failure": "échec du scan",
```

Comment:
For modal titles, we always use upper-case letter for the beginning of the sentence, we may need to print Echec instead of échec